elrond-wasm/src/storage/mappers/linked_list_mapper.rs

```diff
@@ -23,6 +22,38 @@ pub struct Node {
 	pub next: u32,
 }
 
+#[derive(TopEncodeOrDefault, TopDecodeOrDefault, PartialEq, Clone, Copy)]
+pub struct Info {
```

Comment:
I would name this LinkedListMapperInfo.
It is true that we have modules, but there are high chances that this will be exported as-is and lost some of the context.

Reply:
Done.